### ISSUE TITLE
Thoughts on Jack

### ISSUE THREAD

**Issue by aftabahmad3** [vincekoo/MVP#2]
I like that it introduces the new Java 8 features beyond what Retrolambda can do, (Lambdas and method references).

Personally though I feel Jack isn't quite ready for a production environment, it still has a slow build time compared to the current build process.

**Comment by vincekoo** [comment]
I agree with you, I do not feel like Jack is ready for production environment too. Running tests to compare the build process would be a fantastic idea!
